src/components/enterprise-page/data/tests/customerConfig.test.js

```diff
@@ -0,0 +1,46 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import { useEnterpriseCustomerConfig } from '../hooks';
+import { fetchEntepriseCustomerConfig } from '../service';
+
+jest.mock('../service');
+
+// for now let's mock router dom to mock useParams, but perhaps we can
+// pass a default valued arg into the hook to avoid this?
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
```

Comment:
this madness is here because the hook depends on the useParams hook. I could avoid it if we can pass the enterpriseSlug as a defaulted argument to the hook, but not sure if that breaks any design in the application

Reply:
I think it's fine to pass `enterpriseSlug` as an argument to the hook to avoid depending on `useParams` within the hook itself.